processing/src/main/java/io/druid/query/Druids.java

```diff
@@ -462,13 +463,9 @@ public TimeseriesQueryBuilder filters(String dimensionName, String value)
       return this;
     }
 
-    public TimeseriesQueryBuilder filters(String dimensionName, String value, String... values)
```

Comment:
ah I see you're checking this in InDimFilter's constructor. In that case why not keep `value, values...`.